src/apps/chifra/pkg/unchained/UnchainedIndex.multichain.sol

```diff
@@ -0,0 +1,60 @@
+/*
+
+See the other file also -- UnchainedIndexv2.sol
+
+Version 2
```

Comment:
In a future version 3.0, we might want to consider this post related to the gas cost of publishing the hash as a string. https://ethereum.stackexchange.com/questions/61100/how-much-does-it-cost-to-store-each-ipfs-hash-in-ethereum-blockchain/83025. I guess there are cheaper ways gas-wise. But we won't do anything here. I'll make an issue.